chainer/datasets/pickle_dataset.py

```diff
@@ -94,6 +96,11 @@ def __init__(self, reader):
             self._positions.append(position)
 
         self._lock = threading.RLock()
+        mputil.register_after_fork(self, PickleDataset._after_fork)
```

Comment:
`mputil.register_after_fork` is not documented. Please add a TODO such as "Avoid using undocumented feature".